reoptjl/models.py

```diff
@@ -233,10 +233,10 @@ class TIME_STEP_CHOICES(models.IntegerChoices):
         FOUR = 4
 
     timeout_seconds = models.IntegerField(
-        default=420,
+        default=600,
         validators=[
             MinValueValidator(1),
-            MaxValueValidator(420)
+            MaxValueValidator(1200)
```

Comment:
I see no harm in going higher for API users, like 3600 - since harder problems might need more time.

Reply:
I'd be fine with a larger timeout for local/other-server running the API, but I don't want to clog up the NREL servers with super-long running runs. That's actually the main motivation for keeping this reasonably low.

Reply:
Makes sense to me!  Thanks for clarifying.